<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutStationnement</class>
 <widget class="QDialog" name="AjoutStationnement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutStationnement</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>identifiant de la borne stationnement:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>identifiant voie publique :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nom topographique:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>latitude:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de la borne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cote de la rue:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="case_voie">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="case_topo">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="case_cote">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="case_id">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="case_long">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>9999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="case_lat">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999.990000000005239</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="case_numborne">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmer_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Abandonner_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Abandonner</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmer_btn</sender>
   <signal>clicked()</signal>
   <receiver>AjoutStationnement</receiver>
   <slot>valider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Abandonner_btn</sender>
   <signal>clicked()</signal>
   <receiver>AjoutStationnement</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valider()</slot>
 </slots>
</ui>
